### PR TITLE
Add gopher URL

### PR DESCRIPTION
**Pull request by PhilmacFLy**
As Gopher is a fast-growing emerging technology, I find it important that a space can also show his representation in the gopher space on the space API. Now with example and fixed spelling :)

### CODE REVIEW

**Comment by dbrgn** [comment]
Thanks! By the way, you could have simply pushed a fix to the other branch instead of opening a new pull request :slightly_smiling_face:

**Comment by PhilmacFLy** [comment]
As I didn't check it locally but modified in only in github itself, I thought this was the easier method, also with adding the new changes since feburary.

File: 14-draft.json [hunk 1]
```diff
@@ -272,6 +272,9 @@
         "issue_mail": {
           "description": "A separate email address for issue reports (see the <em>issue_report_channels</em> field). This value can be Base64-encoded.",
           "type": "string"
+        },
+        "gopher": {
+          "description": "A URL to find information about the Space in the Gopherspace. Example: gopher://gopher.binary-kitchen.de"
```
Comment (rnestler) [inline 14-draft.json]: Isn't the `"type": "string"` field missing?

Comment (PhilmacFLy) [reply 14-draft.json]: yeah ofc it is m(

**Comment by PhilmacFLy** [comment]
Check again please, should be fixed now

**Comment by rnestler** [comment]
> Check again please, should be fixed now

Indeed. @dbrgn We should probably add CI checks for the syntax of the file.

**Pull request merged**